tensorflow_model_optimization/g3doc/guide/pruning/comprehensive_guide.ipynb

```diff
@@ -0,0 +1,848 @@
+{
```

Comment:
<p>Line 17: the input_shape here will be used in the setup_model() function in the cell below Model.fit, which is causing a problem since the setup_model() expects input_shape=[20].</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/272/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>